packages/terra-show-hide/src/ShowHide.jsx

```diff
@@ -58,6 +59,15 @@ const ShowHide = (props) => {
     ...customProps
   } = props;
 
+  const theme = React.useContext(ThemeContext);
+  const ref = React.useRef(null);
+
+  React.useEffect(() => {
+    if (isOpen && ref?.current) {
+      ref.current.focus();
```

Comment:
Can we add a test for this?
e.g.

https://github.com/cerner/terra-framework/blob/13f7f4c7e2bc71f5a650646bd887d99c730bca48/packages/terra-slide-panel/tests/wdio/slide-panel-spec.js#L185C4-L195

```js
$('#test-slide [aria-label="Panel content area"][aria-hidden="false"]').waitForExist();
browser.keys(['Tab']);
expect($('#close-panel-button').isFocused()).toBeTruthy();
```

Reply:
Added [here](https://github.com/cerner/terra-core/blob/9e8aff2a397aa02be6fda129d3f049612e78d1e0/packages/terra-show-hide/tests/wdio/show-hide-spec.js#L32-L42)